docs/program/extend/custom-components-remotes.md

```diff
@@ -4,21 +4,23 @@ linkTitle: "Custom Components as Remotes"
 weight: 99
 type: "docs"
 tags: ["server", "sdk"]
+aliases:
```

Comment:
Previously in this file there was some info that said:
```
Implementing a custom component allows you to create a new component type (or a new model of an existing type, such as a new `arm` model) that interfaces with `viam-server` using Viam's SDKs.
A custom component is a subclass of a [component](https://python.viam.dev/autoapi/viam/components/component_base/index.html#module-viam.components.component_base) that allows you to register the new component with the `viam-server` as a {{< glossary_tooltip term_id="remote" text="remote" >}} after you spin up an SDK server (which can have one or many custom components).
```

This seemed helpful but I think it's gone missing

Reply:
(https://github.com/viamrobotics/docs/pull/1029#discussion_r1180269301)

Reply:
I agree that this seems helpful but James pointed out that it is inaccurate! The subclass terminology is only true for Python, and the registration is with the SDK not `viam-server`, + "sdk server" is inaccurate

I tried to add more to the example to clarify that component API methods are available & put a sentence in the instructional language tabs clarifying `Each remote server can host one or many custom components.`

Reply:
ok thank you!